client/src/components/ReportSummary.js

```diff
@@ -193,35 +193,33 @@ ReportSummary.propTypes = {
 
 const ReportSummaryRow = ({ report }) => {
   report = new Report(report)
+  const className = `report-${report.getStateForClassName()}`
 
   return (
     <Grid fluid className="report-summary">
       {report.isDraft() && (
-        <p className="report-draft">
+        <p>
+          <span className={className} />
```

Comment:
As the <p><span className={className}> is always the same, I would add the conditionals depending on state only after, to not repeat the rest.

Reply:
No, that would change the logic (not all checks are mutually exclusive).